docs/catalyst_101.md

```diff
@@ -173,76 +176,69 @@ We don't prescribe how each team should organize its work. However, we expect ea
 
-- [**Work Teams**](https://drive.google.com/drive/folders/1CHbOJU6W3r3DE2LEQqbcfdme5DK-qufz): based around the work of a particular client or grant and only exist so long as the client remains a client or the grant remains active.
-- [**Business Teams**](https://drive.google.com/drive/folders/1KuF1sJGQMWmODpD5LhnWVN61Ymbmg9xR): serve ongoing co-op needs.
+- [**Work Teams**](https://drive.google.com/drive/folders/1CHbOJU6W3r3DE2LEQqbcfdme5DK-qufz): Delegated tasks relating to the work of client  grant, or internal technical projects. Right now there are two work
+  teams that handle grant and client work and one work team that
+  handles PUDL-specific infrastructure work.
+- [**Internal Teams**](https://drive.google.com/drive/folders/1KuF1sJGQMWmODpD5LhnWVN61Ymbmg9xR): Serve ongoing, non-technical co-op needs.
 
 Teams may be asked to perform research or create proposals based on the outcome of conversations in PUDL Huddle or Board meetings. Teams will have their own slack channels. Any Catalyst member can join any of the teams’ slack channels so that they can post inquiries or ask favors of specific teams. Teams do not have a maximum or minimum number of members. Teams may have only one member or be composed of everyone in the organization (though neither is recommended).
 
 All co-op activities are in the domain of a particular team.
 
 #### Current Work Teams
 
-- **RMI**
-- **Inframundo** (Sloan)
-- **DBCP**: Down Ballot Climate Project
-- **WEIB**: Western Interstate Energy Board
-- **CCAI**: Climate Change AI
+- **Caracal** ([project board](https://github.com/orgs/catalyst-cooperative/projects/9/views/22))
+- **Margay** ([project board](https://github.com/orgs/catalyst-cooperative/projects/9/views/24))
+- **Inframundo** ([project board](https://github.com/orgs/catalyst-cooperative/projects/9/views/6))
 
-#### Business Teams
+(internal-teams)=
 
-- **Admin**
-- **Internal Management**
-- **Community Development**
-- **Grant Development**
-- **Client Development**
-- **Hiring & New Employee Management**
+#### Internal Teams
+
+- **Admin**: Run taxes, invoicing, payroll.
+- **Internal Management**: Curate PM tools and processes, track internal capacity, HR, documentation of processes.
+- **Community Development**: Build community around PUDL development and data use. Improve user experience, communicate our mission to the world, and interact with users and contributors.
+- **Grant Development**: Identify grant opportunities that are relevant to PUDL and
+  Catalyst.
+- **Business Development**: Identify opportunities and develop new relationships that lead to revenue-generating, mission-aligned client projects. Maintain relationships with existing clients.
+- **Hiring & New Employee Management**: Ad hoc team for hiring and onboarding new
```

Comment:
We're missing PUDL Pat here, should we include it?

Reply:
I would say no. We can add that later if it becomes more of a thing.